.devcontainer/Dockerfile

```
@@ -7,9 +7,4 @@ RUN echo 'export PATH="/usr/lib/ccache:$PATH"' | tee -a /home/vscode/.bashrc
 # build variables
 #RUN echo 'export BUILDDIR="/workspaces/r-dev-env/bin/R"' >> ~/.bashrc 
 #RUN echo 'export TOP_SRCDIR="/workspaces/r-dev-env/Downloads/R"' >> ~/.bashrc 
```

TOP_SRCDIR needs changing here too

```suggestion
#RUN echo 'export TOP_SRCDIR="/workspaces/r-dev-env/svn/R-devel"' >> ~/.bashrc 
```